docs/v3_services.rst

```diff
@@ -0,0 +1,249 @@
+  instance as follows:
+
+  - Use the Source desktop shortcut to connect to the *Source Interface* and 
+    verify that the new 56-character address is present in the Tor Browser 
+    address bar
+  - Use the Journalist desktop shortcut to connect to the *Journalist Interface*
+    and verify that the new 56-character address is present in the Tor Browser 
+    address bar
+  - Use the commands ``ssh app`` and ``ssh mon`` in a terminal to verify 
+    SSH access to the *Application* and *Monitor Servers*.
+
+- Finally, back up the instance and *Admin Workstation* USB.
+
+(Optional) enabling HTTPS
+^^^^^^^^^^^^^^^^^^^^^^^^^
+If your instance serves the *Source Interface* over HTTPS, and you plan to 
+continue using HTTPS with v3 onion services, you'll need to provision a 
+new certificate for the new v3 address.
+
+You'll find the new *Source Interface* address in the file:
+
+.. code-block:: none
+ 
+  ~/Persistent/securedrop/install_files/ansible-base/app-sourcev3-ths
+
+Follow the instructions in :doc:`HTTPS on the Source Interface <https_source_interface>`
+to provision and install the new certificate. 
+
+
+Updating Workstation USBs
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you chose to keep v2 enabled, *Admin* and *Journalist Workstations* that have
+not yet been updated will still be able to connect to the v2 onion services. Even
+so, you should update all workstations to use v3 services as soon as possible.
+
+Journalist Workstation:
+~~~~~~~~~~~~~~~~~~~~~~~
+
+ - In the *Admin Workstation* used to enable v3 onion services, copy the 
+   following files to a *Transfer Device*:
+
+   .. code-block:: none
+
+     ~/Persistent/securedrop/install_files/ansible-base/app-sourcev3-ths
+     ~/Persistent/securedrop/install_files/ansible-base/app-journalist.auth_private
+
+   Then, boot into the *Journalist Workstation* to be updated.
+
+ - In the *Journalist Workstation*,  first ensure that the 
+   SecureDrop application code has been updated to the latest version, using
+   either the GUI updater or the ``./securedrop-admin update`` command.
+ 
+ - Copy the ``app-sourcev3-ths`` and ``app-journalist.auth_private`` files from
+   the *Transfer Device* to ``~/Persistent/securedrop/install_files/ansible-base``.
+ 
+ - Open a terminal and run ``./securedrop-admin tailsconfig`` to update the 
+   SecureDrop desktop shortcuts.
+
+ - Verify that the new 56-character addresses are in use by visiting the *Source*
+   and *Journalist Interfaces* via the SecureDrop desktop shortcuts.
+
+ - Securely wipe the files on the *Transfer Device*.
+
+Admin Workstation:
+~~~~~~~~~~~~~~~~~~
+
+ - In the *Admin Workstation* used to enable v3 onion services, copy the 
+   following files to a *Transfer Device*:
+
+   .. code-block:: none
+
+     ~/Persistent/securedrop/install_files/ansible-base/app-sourcev3-ths
+     ~/Persistent/securedrop/install_files/ansible-base/app-journalist.auth_private
+     ~/Persistent/securedrop/install_files/ansible-base/app-ssh.auth_private
```

Comment:
should we denote in this block something like `(if SSH over Tor is enabled)` for this line and the line below?

Reply:
:white_check_mark: